apps/avifdec.c

```diff
@@ -115,6 +117,14 @@ int main(int argc, char * argv[])
             } else if (jpegQuality > 100) {
                 jpegQuality = 100;
             }
+        } else if (!strcmp(arg, "--png-compress")) {
+            NEXTARG();
+            pngCompressionLevel = atoi(arg);
+            if (pngCompressionLevel < 0) {
+                pngCompressionLevel = 0;
```

Comment:
Would it make sense to print something here if you clamp the value?

Reply:
Or fail with an "invalid PNG compression level" message.